src/WebBrowser-Core.package/WebBrowser.class/class/webbrowserClassForPlatform.st

```diff
@@ -0,0 +1,8 @@
+private - accessing
+webbrowserClassForPlatform
+
+	Smalltalk os isUnix ifTrue: [ ^WBUnixWebBrowser ].
+	Smalltalk os isWindows ifTrue: [ ^WBWindowsWebBrowser ].
+	(Smalltalk os isMacOS or: [ Smalltalk os isMacOSX ])  
+		ifTrue: [ ^WBMacWebBrowser ].
```

Comment:
Thank you! I only commented from reading the code in github so I missed the WebBrowser class.